simapp/upgrades.go

```diff
@@ -27,6 +27,9 @@ func (app SimApp) RegisterUpgradeHandlers() {
 			// dedicated x/consensus module.
 			baseapp.MigrateParams(ctx, baseAppLegacySS, &app.ConsensusParamsKeeper)
 
+			// Note: this migration is optional,
+			// You can include x/gov proposal migration documented at UPGRADING.md(https://github.com/cosmos/cosmos-sdk/blob/main/UPGRADING.md)
```

Comment:
Nit: does this link is supported to be markdown or misses a space between the parentheses?